src/scss/constants.scss

```diff
@@ -36,7 +36,7 @@ $danger-gradient: linear-gradient(
 );
 $admin-gradient: linear-gradient(141deg, #5a35ad 0%, #26144f 100%);
 
-$map-sidebar-width: 20%;
+$sidebar-width: 360px;
```

Comment:
As mentioned in another comment in this review, perhaps instead of a static `px` width here, we might want to make it responsive? Probably with a minimum width as well.

(cc // @mike-lvov @mcflyok FYI: Thoughts?)

Reply:
left as fixed value, updated styles for Chatsidebar